tasks/eth/008-presigned-pause/tx/draft-92.json

```diff
@@ -0,0 +1,10 @@
+{
+  "chain_id": "1",
+  "rpc_url": "https://eth-mainnet.g.alchemy.com/v2/xs6fOA9axQtr6xxN0cvaeBruNrpZhrqo",
+  "created_at": "2024-06-04T13:49:03-04:00",
+  "safe_addr": "0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A",
+  "safe_nonce": "92",
+  "target_addr": "0x5dc91d01290af474ce21de14c17335a6dee4d2a8",
+  "script_name": "PresignPauseFromJson_eth_008",
+  "data": ""
+}
```

Comment:
I was worried that this `data` field is empty, but I checked that it was the same in previous PSP ceremonies: 

https://github.com/ethereum-optimism/superchain-ops/blob/maur/eth/008-psp/tasks/eth/007-presigned-pause/tx/draft-92.json#L9